src/clients/types.ts

```diff
@@ -0,0 +1,20 @@
+export type Client = {
+  id: number;
+  company_name: string;
+  active: boolean;
+  username: string;
```

Comment:
I think we should split it up to have the types for create/update and read.

Reply:
True, I was considering this as well.

I am not sure how much sense it makes, given that our postman collection and the docs on https://api.timetac.com only have the `read` endpoint.

I can try to reconstruct this by relying directly on the backend implementation, but I'm not sure how accurately I will get this.

What do you say, is it worth the time?

Reply:
If we only have read, leave it for now.